cfme/fixtures/merkyl.py

```diff
@@ -68,6 +69,33 @@ def search_log(self, needle, log_name):
         else:
             return False
 
+    def search_log_with_regex(self, expr, log_name, _flag):
```

Comment:
**REQUIRED** use actual regex flags instea of magical constants

Reply:
Done.